audiofile/core/io.py

```diff
@@ -53,15 +58,29 @@ def convert_to_wav(
             but cannot be found
         RuntimeError: if ``file`` is missing,
             broken or format is not supported
+        RuntimeError: if ``infile`` would need to be overwritten
+            and ``overwrite`` is ``False``
 
     Examples:
-        >>> path = convert_to_wav('stereo.flac', 'stereo.wav')
+        >>> path = convert_to_wav('stereo.flac')
         >>> os.path.basename(path)
         'stereo.wav'
 
     """
     infile = audeer.safe_path(infile)
-    outfile = audeer.safe_path(outfile)
+    if outfile is None:
+        outfile = audeer.replace_file_extension(infile, 'wav')
+    else:
+        outfile = audeer.safe_path(outfile)
+    if (
+            infile == outfile
+            and not overwrite
+    ):
+        raise RuntimeError(
+            f"'{infile}' would need to be overwritten. "
```

Comment:
```suggestion
            f"'{infile}' would be overwritten. "
```

Reply:
I adjusted the error message here and in the test accordingly with my last commit.